README.md

```diff
@@ -1,55 +1,79 @@
-6. The Swift Package template includes a Swift Package Index configuration file to automatically build the package and [host the documentation on the Swift Package Index website](https://blog.swiftpackageindex.com/posts/auto-generating-auto-hosting-and-auto-updating-docc-documentation/). Adjust the `.spi.yml` file to include all targets that you want to build documentation for. You can follow the [instructions of the Swift Package Index](https://swiftpackageindex.com/add-a-package) to include your Swift Package in the Swift Package Index. You can link to the [API documentation](https://swiftpackageindex.com/StanfordBDHG/SwiftPackageTemplate/documentation) from your README file.
-7. Adjust the CITATION.cff file to amend information about the new Swift Package ([learn more about CITATION files on GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)) and [register the Swift Package on Zenodo](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content). 
+## Overview
 
+The Spezi License module provides a quick way to inform users about the tools and packages you have leveraged in your project including their license information.
+You use the ``ContributionsList`` abstraction within your views to visualize a list of all Swift package dependecies used in your Xcode project.
 
-## Installation
+This package builds on Felix Hermann's' [SwiftPackageList](https://github.com/FelixHerrmann/swift-package-list) library under the hood.
 
-The project can be added to your Xcode project or Swift Package using the [Swift Package Manager](https://github.com/apple/swift-package-manager).
 
-**Xcode:** For an Xcode project, follow the instructions on [adding package dependencies to your app](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app).
+## Setup
 
-**Swift Package:** You can follow the [Swift Package Manager documentation about defining dependencies](https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md#defining-dependencies) to add this project as a dependency to your Swift Package.
+### 1. Add Spezi License and Swift Package List as a Dependency.
+
+You need to add the SpeziLicense and [SwiftPackageList](https://github.com/FelixHerrmann/swift-package-list) Swift package to
+[your app in Xcode](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app#) or
+[Swift package](https://developer.apple.com/documentation/xcode/creating-a-standalone-swift-package-with-xcode#Add-a-dependency-on-another-Swift-package).
+
+### 2. Add the Run Script Phase to your Xcode Project
+
+Follow the "Run Script Phase" setup steps of the [SwiftPackageList](https://github.com/FelixHerrmann/swift-package-list?tab=readme-ov-file#run-script-phase) documentation.
```

Comment:
As demonstrated in https://github.com/StanfordSpezi/SpeziTemplateApplication/pull/75 you can use the functionality just using the build time plugin; this also does not require someone to install the swift package list tool locally. I would therefore suggest to remove this and make this README a bit more compact.

Reply:
you're right! Removed the script and the 2. step entirely. Nonetheless, to have access to the build time plugin, one still needs to add the SwiftPackageList dependency to the app.

Reply:
Thank you @NikolaiMadlener!